app/js/states/actions/actions.jade

```diff
@@ -30,17 +31,51 @@
                         i.fa.fa-square.inline-right(ng-class="category.id + '-color'", aria-hidden="true")
                         span.desc(translate) {{category.id | titlecase}}
                         span.count &nbsp;({{category.count}})
-        .col-md-5
+        .col-md-6
+          card.widget.widgetIncidentSummary(id="widgetIncidentSummary", ng-show='incidentCount > 0')
+            card-header.gray
+              h4(translate) Incident summary
+            card-content.flexgrid.equalheights
+              .row
+                .col-sm-3
+                  #countComparison.countComparison
+                    .count
+                      .num {{incidentCount}}
+                      .desc(translate) Incidents
+                    .count
+                      .num {{incidentSystemCount}}
+                      .desc(translate) Systems
+                .col-sm-9.border-left
+                  .icon-inline
+                    i.fa.fa-bullseye
+                    p(translate) An&nbsp;
+                      strong incident&nbsp;
+                      | is an issue that is currently or has already affected systems. {{incidentCount}} of {{stats.rules.total}} issues are INCIDENTS affecting {{incidentSystemCount}} systems. It is imperative that these incidents be addressed to ensure optimal performance.
```

Comment:
translate-plural and friends should be used here to prevent **"affecting 1 systems"**